.gitignore

```diff
@@ -58,6 +58,7 @@ docs/_build/
 troposphere.key
 troposphere/settings/local.py
 troposphere/assets
+troposphere/tropo-static
```

Comment:
thats what assets is used for?
https://github.com/iPlantCollaborativeOpenSource/troposphere/pull/419